src/main.java

```diff
@@ -0,0 +1,68 @@
+public class main {
```

Comment:
I would have put this at the bottom of your inventory file, but this looks great!